frappe/database/mariadb/setup_db.py

```diff
@@ -69,10 +64,15 @@ def drop_user_and_database(db_name, root_login, root_password):
 def bootstrap_database(db_name, verbose, source_sql=None):
 	import sys
 
+<<<<<<< HEAD
 	frappe.connect(db_name=db_name)
 	if not check_database_settings():
 		print("Database settings do not match expected values; stopping database setup.")
 		sys.exit(1)
+=======
+	frappe.connect()
+	check_compatible_versions()
+>>>>>>> ed01fc3b26 (feat: don't require editing MariaDB configuration to setup frappe (#25609))
```

Comment:
```suggestion
	frappe.connect(db_name=db_name)
	check_compatible_versions()
```

Suggestion as per @Thatoo's PR https://github.com/frappe/frappe/pull/25759  
Original PR for this backport: https://github.com/frappe/frappe/pull/25609

Note: `check_database_settings()` is not used anymore because the settings are controlled by Frappe, instead of just blindly trusting the global DB config (if I understood the PR mentioned above).